pymeasure/display/Qt.py

```diff
@@ -24,13 +24,18 @@
 
 import logging
 
-from pyqtgraph.Qt import QtGui, QtCore, loadUiType
+from pyqtgraph.Qt import QtGui, QtCore, QtWidgets, loadUiType
 
 log = logging.getLogger(__name__)
 log.addHandler(logging.NullHandler())
 
 QtCore.QSignal = QtCore.Signal
 
+# Should be removed when PySide2 provides QtWidgets.QApplication.exec() or when support for PySide2
+# is dropped
+if not hasattr(QtWidgets.QApplication, 'exec'):
+    QtWidgets.QApplication.exec = QtWidgets.QApplication.exec_
```

Comment:
that looks like a straightforward patch to me, although it would be good to reference an upstream issue and/or documentation on the background of this to the comment. This would probably make it easier to pick this up/recheck in the future.
I don't know about any potential ramifications of this. I gather we cannot have been the first ones running into the issue, though.

Reply:
Just googling this issue, you find the explanation quite clear here https://stackoverflow.com/questions/22610720/should-i-use-app-exec-or-app-exec-in-my-pyqt-application.
Essentially, this is related to the fact that `exec` was a reserved keyword in python2 and so people was forced to use `exec_`. This constraint is no longer present in python3 and so `PyQt` people added support for `exec`, but `PySide2` people didn't.

Reply:
I've added a link to the comment, though I could not find an upstream issue as I have the impression that this is not something that is considered an "issue" in PySide2